tests/data/hyper_data.json

```diff
@@ -0,0 +1 @@
2773332088149}, {"metric": 0.3556452256494941}, {"metric": 0.35348163370962}, {"metric": 0.3518306360094952}, {"metric": 0.34886203800770393}, {"metric": 0.34564913993272545}, {"metric": 0.3415958491988287}, {"metric": 0.3377696926921215}, {"metric": 0.33484552607069007}, {"metric": 0.33247218704435344}, {"metric": 0.3300964805382674}, {"metric": 0.3277299851845919}, {"metric": 0.32551497300819293}, {"metric": 0.3234565703265598}, {"metric": 0.3214969299238092}, {"metric": 0.3193841280647433}, {"metric": 0.31708120097742537}, {"metric": 0.3147812297903259}, {"metric": 0.31261269844159095}, {"metric": 0.3110467163866468}, {"metric": 0.3093930810887473}, {"metric": 0.3079395944367787}, {"metric": 0.30631853410532256}, {"metric": 0.3046236352612792}, {"metric": 0.3029537253818512}, {"metric": 0.3013906497787554}, {"metric": 0.2999159486971349}, {"metric": 0.29850350558248184}, {"metric": 0.297391015322096}, {"metric": 0.29609008531306313}, {"metric": 0.2950517120468839}, {"metric": 0.2939231772299107}, {"metric": 0.29302975391830427}, {"metric": 0.29205312779845394}, {"metric": 0.29102237385688284}, {"metric": 0.29025719692056245}, {"metric": 0.28950336396661197}, {"metric": 0.28888051875787013}, {"metric": 0.2882786251459921}, {"metric": 0.28779032821163614}, {"metric": 0.2873429229725943}, {"metric": 0.28702079307444056}, {"metric": 0.2866118697466162}, {"metric": 0.2862814199903141}, {"metric": 0.28582874840149763}, {"metric": 0.28559691782488783}, {"metric": 0.28528204034300714}, {"metric": 0.2850683870112878}, {"metric": 0.284710470555042}, {"metric": 0.28442339132104083}, {"metric": 0.28419927947122064}, {"metric": 0.2839387012458933}, {"metric": 0.28374112405791546}, {"metric": 0.28345509505085276}, {"metric": 0.2833843302063161}, {"metric": 0.2832073398147285}, {"metric": 0.28318910702744815}, {"metric": 0.283024819831691}, {"metric": 0.282965968706981}, {"metric": 0.2827156820121598}, {"metric": 0.2825203493642208}, {"metric": 0.2824398029982966}, {"metric": 0.2824404728992157}, {"metric": 0.2824042248940179}, {"metric": 0.2821967394973826}, {"metric": 0.28207524703345876}, {"metric": 0.2819687940675698}, {"metric": 0.2819997219133576}, {"metric": 0.2819313305314162}, {"metric": 0.2818489886859193}, {"metric": 0.281559712378093}, {"metric": 0.28125874715927196}, {"metric": 0.2811480991747629}, {"metric": 0.28116257787277144}, {"metric": 0.2812716301552804}, {"metric": 0.2812077574053755}, {"metric": 0.28107653161365254}, {"metric": 0.28082713578706836}, {"metric": 0.280652858370934}, {"metric": 0.2806063368255118}, {"metric": 0.280537679581214}, {"metric": 0.2804418960227811}, {"metric": 0.28036433466818994}, {"metric": 0.28042510048674646}, {"metric": 0.28036179278562623}, {"metric": 0.28026646974357367}, {"metric": 0.2801942257179042}, {"metric": 0.2802555414381709}, {"metric": 0.28023845988646534}, {"metric": 0.2801453290941737}], [{"metric": 0.22905999604424632}, {"metric": 0.2958377293139273}, {"metric": 0.3181781941906532}, {"metric":
```

Comment:
can we make this a yaml file and at the top add a comment that describes what this is, and any remarkable properties about the data (minimum at very early step)